lib/streamlit/config.py

```diff
@@ -810,45 +810,30 @@ def _s3_secret_access_key():
 
 _create_option(
     "theme.primaryColor",
-    description="""
-        Used to style primary interface elements. It's the color displayed
-        most frequently across your app's screens and components. Examples of
-        widgets using this color are st.slider and st.checkbox.
-        """,
-)
-
-_create_option(
-    "theme.secondaryColor",
-    description="""
-        Used to style secondary interface elements. It provides more ways to
-        accent and distinguish your app. Having it is optional.
-        """,
+    description="Primary accent color for interactive elements.",
 )
 
 _create_option(
     "theme.backgroundColor",
-    description="Background color for the main container.",
+    description="Background color for the main content area.",
 )
 
 _create_option(
     "theme.secondaryBackgroundColor",
-    description="""
-        Used as the background for most widgets. Examples of widgets with this
-        background are st.sidebar, st.text_input, st.date_input.
-    """,
+    description="Background color used for the sidebar and most interactive widgets.",
```

Comment:
most **of the** interactive widgets? (if you decide to change, I've seen this text in a few places)

Reply:
I might be wrong here - feel free to ignore this comment.

Reply:
I think the two are grammatically equivalent, so I'll leave this as-is as the wording came from product. I also think there will probably be one last pass on the copy before release where any small tweaks can be made